Jenkinsfile

```diff
@@ -47,7 +47,8 @@ pipeline {
         stage('Prepare Release YAML') {
             steps {
                 script {
-                    releaseData = edgeXRelease.collectReleaseYamlFiles()
+                    sh 'env | sort'
+                    releaseData = edgeXRelease.collectReleaseYamlFiles('release/*.yaml', 'origin/release')
```

Comment:
I had to add `origin/release` here because it was not finding any diff's otherwise.

Reply:
It defaults to just `release` ... I wonder why we get `origin/release` sometimes

Reply:
Yeah, I had to put `origin/` for it to finally pick up